packages/cli/cloud/package.json

```diff
@@ -46,7 +46,7 @@
   },
   "dependencies": {
     "@strapi/utils": "4.25.8",
-    "axios": "1.6.0",
+    "axios": "1.7.4",
```

Comment:
Should we use version `^1.7.4` which will handle the update automatically for us in the future?

Reply:
Hi @Marc-Roig thank you for your reply.
Can you please clarify the issue you experienced with Axios versioning. 

Is it the issue on Axios end when we do minor upgrade? Because if they follow Semver, it should not causing issue. But who knows, could be something that I missing.

Reply:
Absolutely! We have never encountered any issue in particular with Axios, but unfortunately and in practice, some packages do introduce breaking changes even in patch updates, so we prefer to stay on the safe side an apply this rule for all packages.